### PR TITLE
DOC removing pyplot_annotate.py

### PR DESCRIPTION
**Pull request by NelleV**
Remove duplicate example.

### CODE REVIEW

File: examples/pyplots/annotation_basic.py [hunk 1]
```diff
@@ -17,6 +19,4 @@
 ax.annotate('local max', xy=(2, 1), xytext=(3, 1.5),
             arrowprops=dict(facecolor='black', shrink=0.05),
             )
-
-ax.set_ylim(-2,2)
-plt.show()
```
Comment (dstansby) [inline examples/pyplots/annotation_basic.py]: Shouldn't all the examples have `plt.show()` at the end so they can be run as `python example.py`?

Comment (NelleV) [reply examples/pyplots/annotation_basic.py]: Feel free to fix this.

**Comment by choldgraf** [comment]
LGTM, @NelleV I saw that one of the tests didn't pass, was that related to this PR? If not, I added plt.show to the end and am happy to merge

**Comment by NelleV** [comment]
doesn't seem to be related.

**Comment by tacaswell** [comment]
Thanks @NelleV !

**Comment by choldgraf** [comment]
Thanks @NelleV ! Removing redundant files is the git equivalent to cleaning out earwax...strangely satisfying :-D

**Pull request merged**